sdk/search/azure-search-documents/src/main/java/com/azure/search/documents/FieldBuilder.java

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.search.documents;
+
+import com.azure.search.documents.models.ComplexSearchField;
+import com.azure.search.documents.models.SimpleDataType;
+import com.azure.search.documents.models.SimpleSearchField;
+
+/**
+ * Helper to build search field
+ */
+public class FieldBuilder {
+    public static SimpleSearchField getSimpleSearchField() {
```

Comment:
Why does this builder exist when we can just run `new SimpleSearchField()`?

Reply:
I am having this like a factory.
Just want to let customer know the type of Field they can use. Naming of the class is flexible to change.

Reply:
This class shouldn't be kept - as Connie notes it doesn't appear to offer any additional value to the user.

Reply:
I am working on rewrite the FieldBuilder. Thanks for comments!